React 19.0 incompatibility

the command
 **npm install --save @u-wave/react-vimeo**
create error as follows.

**npm error code ERESOLVE  
npm error ERESOLVE unable to resolve dependency tree  
npm error
npm error While resolving: curved-centauri@0.0.1
npm error Found: react@19.0.0
npm error node_modules/react
npm error   react@"^19.0.0" from the root project
npm error
npm error Could not resolve dependency:
npm error peer react@"^0.14.0 || ^15.0.0 || ^16.0.0 || ^17.0.0 || ^18.0.0" from @u-wave/react-vimeo@0.9.11
npm error node_modules/@u-wave/react-vimeo
npm error   @u-wave/react-vimeo@"*" from the root project**

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -39,7 +39,7 @@ jobs:
       fail-fast: false
       matrix:
         node-version: [16.x, 18.x, 20.x]
-        react-version: [17.x, 18.x, beta]
+        react-version: [17.x, 18.x, 19.x]
         include:
           - node-version: 14.x
             react-version: 16.0.0

--- a/package.json
+++ b/package.json
@@ -33,13 +33,13 @@
   },
   "homepage": "https://github.com/u-wave/react-vimeo#readme",
   "dependencies": {
-    "@types/react": "^17.0.0 || ^18.0.0",
+    "@types/react": "^17.0.0 || ^18.0.0 || ^19.0.0",
     "@types/vimeo__player": "^2.10.0",
     "@vimeo/player": "^2.16.4",
     "prop-types": "^15.7.2"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0 || ^16.0.0 || ^17.0.0 || ^18.0.0"
+    "react": "^0.14.0 || ^15.0.0 || ^16.0.0 || ^17.0.0 || ^18.0.0 || ^19.0.0"
   },
   "devDependencies": {
     "@babel/core": "^7.12.10",

